src/constants/apiPermissions.js

```diff
@@ -1,8 +1,18 @@
-// const roles = require('./roles')
+const roles = require('./roles')
 
 module.exports = {
   User: {
-
+    getAll: [roles.SUPER],
+    getById: [ roles.SUPER, roles.OWNER ],
+    create: [ roles.SUPER ],
+    update: [ roles.ADMIN, roles.OWNER ],
+    delete: [ roles.SUPER ],
+    fields: {
```

Comment:
maybe we need to add `createdAt` and `updatedAt` fields restrictions? (I didn't do this on mine either)

(Or maybe Mongoose won't allow you to edit those? But I assume it will)

Reply:
yeah I dunno, I think this is ok for now?

Reply:
as in, letting admin's change it is fine

Reply:
owner can change though

Reply:
yeah true you reckon we should add them to all the routes for super only then? it's not hard to do I guess

Reply:
ahh just leave it for now, it's not like we're using them anywhere yet.